src/backend/distributed/executor/multi_utility.c

```diff
@@ -77,9 +83,12 @@ static void ErrorIfDistributedRenameStmt(RenameStmt *renameStatement);
 /* Local functions forward declarations for helper functions */
 static void CreateLocalTable(RangeVar *relation, char *nodeName, int32 nodePort);
 static bool IsAlterTableRenameStmt(RenameStmt *renameStatement);
-static void ExecuteDistributedDDLCommand(Oid relationId, const char *ddlCommandString);
-static bool ExecuteCommandOnWorkerShards(Oid relationId, const char *commandString,
-										 List **failedPlacementList);
+static void ExecuteDistributedDDLCommand(Oid relationId, const char *ddlCommandString,
+										 bool isTopLevel);
+static void SwitchTo2PCForTransaction();
```

Comment:
Should have void between the brackets.

This function is the equivalent of SET LOCAL citus.multi_shard_commit_protocol TO '2pc'. I think it might make sense to make that more explicit in the name, e.g. SetLocalCommitProtocolTo2PC.